tools/test/topology/test-multi-playback.m4

```diff
@@ -68,19 +68,19 @@ DAI_ADD(sof/pipe-dai-playback.m4,
 
 # playback DAI is SSP TEST_DAI_PORT using 2 periods
 # Buffers use s24le format, with 48 frame per 1000us on core 0 with priority 0
-DAI_ADD(sof/pipe-dai-playback.m4,
-	2, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
-	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
-	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
-#DAI_ADD_SCHED(sof/pipe-dai-sched-playback.m4,
+#DAI_ADD(sof/pipe-dai-playback.m4,
 #	2, TEST_DAI_TYPE, TEST_DAI_PORT, TEST_DAI_LINK_NAME,
 #	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
-#	1000, 1, 0, SCHEDULE_TIME_DOMAIN_TIMER,
-#	PIPELINE_PLAYBACK_SCHED_COMP_1)
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+DAI_ADD_SCHED(sof/pipe-dai-sched-playback.m4,
+	2, TEST_DAI_TYPE, 2, SSP2-Codec,
+	PIPELINE_SOURCE_2, 2, TEST_DAI_FORMAT,
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER,
+	PIPELINE_PLAYBACK_SCHED_COMP_1)
 
 # connect pipelines together
 SectionGraph."pipe-sof-second-pipe" {
-        index "0"
+        index "2"
```

Comment:
I found that this index will determine the order of this routing dapm(PIPELINE_SOURCE_2, PIPELINE_CROSSOVER_1) among the whole m4 file.

My understanding is, index=2 will make this routing handled right after the graph of the second pipeline, is it?

Reply:
I have never used the index field, @lgirdwood can you explain how it is used?